pkg/config/config.go

```diff
@@ -879,7 +871,26 @@ func InitConfig(config Config) {
 
 	// Process agent
 	config.SetKnown("process_config.dd_agent_env")
-	config.SetKnown("process_config.enabled")
+	config.BindEnvAndSetDefault("process_config.process_collection.enabled", false, "DD_PROCESS_COLLECTION")
```

Comment:
What do you think about moving this to a separate file in the `pkg/config` package?

Reply:
Yeah we could have our own file!